### ISSUE TITLE
[C++ front end] how to use clamp to clip gradients?

### ISSUE THREAD

**Issue by ZhuXingJune** [pytorch/pytorch#19098]
## ❓ Questions and Help
hi,  I wonder if this could clip the gradients: 

for(int i=0; i<net.parameters().size(); i++)
		{
			net.parameters().at(i).grad() = torch::clamp(net.parameters().at(i).grad(), -GRADIENT_CLIP, GRADIENT_CLIP);			
		}
optimizer.step();

I found it doesn't seem to work, and I still got large output.
How can I use the "clamp“ correctly?
### Please note that this issue tracker is not a help form and this issue will be closed.

We have a set of [listed resources available on the website](https://pytorch.org/resources). Our primary means of support is our discussion forum:

- [Discussion Forum](https://discuss.pytorch.org/)
